src/mimemail.erl

```diff
@@ -746,14 +754,52 @@ guess_best_encoding(Body) ->
 
 	%based on the % of printable characters, choose an encoding
 	if
-		Percent == 100 ->
-			<<"7bit">>;
 		Percent > 80 ->
 			<<"quoted-printable">>;
 		true ->
 			<<"base64">>
 	end.
 
+%% https://tools.ietf.org/html/rfc2045#section-2.7:
```

Comment:
now there are much more strict requirements to allow body to be sent as is:

no lines longer than 998 bytes
no bare CR or LF
no 0 bytes or bytes above 127